user-guide/Advanced_Functionality/Swarming/Tutorials/EnableSwarmingTutorial.md

```diff
@@ -29,138 +37,128 @@ To deploy the package:
 
-   ![AlarmID Usage](~/user-guide/images/Swarming_Tutorial_Enable_AlarmID_Usage.png)
+   This check can take up to several minutes, depending on the number of scripts and connectors in the system.
 
-   In order for Swarming to work, this alarm identifier has been updated which is a breaking change.
-   To prevent possible issues as much as possible, there is a check to analyze your scripts automatically for problems.
-   Unfortunately it's not possible to scan dependencies (e.g. nugets), so these might still contain outdated references.
-   If this would be the case, running these scripts on a Swarming system would throw a clear error message.
+   ![Alarm ID usage](~/user-guide/images/Swarming_Tutorial_Enable_AlarmID_Usage.png)
 
-   Press the **Analyze** and wait for a bit until the results appear. This can take a while, up to several minutes depending on how many scripts and protocols you have.
+   > [!NOTE]
+   > It is not possible to scan dependencies (e.g. NuGets) with this script, so these may still contain outdated references. However, if this should be the case, running these scripts on a system where Swarming is enabled will throw a clear error message.
 
-   ![AlarmID Usage Problem](~/user-guide/images/Swarming_Tutorial_Enable_AlarmID_Usage_Problem.png)
+1. Take a look at the results of the analysis.
 
-   After analysis is done, the **Mask Critical Alarms** script should appear as one of the found problems.
-   This script takes as input a reference to an element and will mask all critical alarms of this element.
-   It does so by using a **SetAlarmStateMessage** and includes an identifier reference to the alarm it should mask.
-   This identifier still uses the old style of **DataMinerID/AlarmID** integer combination. This is no longer correct and will need a **TreeID** instead.
-   If you were to open this script via DIS in Visual Studio, you would see an obsolete warning.
-   You can also see this if you open the script via the Automation app and press the **Validate** button to check for errors and warnings.
+   ![Alarm ID usage problem](~/user-guide/images/Swarming_Tutorial_Enable_AlarmID_Usage_Problem.png)
+
+   The *Mask Critical Alarms* script will be displayed as one of the problems that have been found. This script takes as input a reference to an element and will mask all critical alarms of this element. It does so by using a *SetAlarmStateMessage* and includes an identifier reference to the alarm it should mask, but this identifier still uses the old style of referring to an alarm, i.e. a **DataMiner ID/Alarm ID** integer combination. This is must be changed to a **tree ID** instead.
+
+1. Open the *Mask Critical Alarms* script in the Automation app, scroll down, and click *Validate* to check for errors and warnings.
+
+   You will see that a warning is detected. If you were to open this script via DIS in Visual Studio, you would also see an obsolete warning.
```

Comment:
perhaps we should mention this only happens if you target the *Skyline.DataMiner.Dev.Automation* package version 10.5.1 or higher